adr/0010-release-process.md

```diff
@@ -0,0 +1,22 @@
+# Release Process
+
+Date: 2018-12-03
+
+## Status
+
+Draft
+
+## Context
+
+eLife has some projects that are published as packages, e.g. for Python projects on pypi.
+
+Specific details may vary from project to project. For example one project may just pushed to pypi, another to Docker Hub, or even both. Some artefacts may just be pushed for releases, others on commit.
+
+## Decision
+
+A new release will be indicated by a new git tag.
+Jenkins will trigger on a new tag to, beyond building and running tests, also push the release out (e.g. pypi, Docker Hub, etc.).
+
```

Comment:
This doesn't prescribe how to deal with `master` and `latest` and whether a tag is always also pushed as something like that. Out of scope for this ADR, so I'll rename it to be more precise in scope. And add consequences.